lib/core/src/lib/viewer/components/viewer.component.scss

```diff
@@ -154,4 +153,15 @@
             display: flex;
         }
     }
+
+    &-toolbar-actions,
+    .adf-viewer-close-button {
+        #{$mat-icon} {
+            margin-top: -10px;
```

Comment:
this is a very rude fix, you are just hammering all icons to be in the negative space to cover the layout problem... it's better finding and fixing why those are layouted incorrectly